tests/tacacs/test_ro_user.py

```diff
@@ -16,7 +16,7 @@
 
 def ssh_remote_run(localhost, remote_ip, username, password, cmd):
     res = localhost.shell("sshpass -p {} ssh "\
-                          "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "\
```

Comment:
This is not the root cause.

UT failed because this:
2021-09-23T04:55:06.5524169Z E       Permission denied, please try again.